### DISCUSSION TITLE
@arichornlover What happen to your Alt?

### DISCUSSION THREAD

**Discussion by yodaluca23** [arichornlover/uYouEnhanced#788] [General]
Was going through some links, and you Alt's GitHub page https://github.com/arichornloverALT is 404 page not found... Is something happening? Or are you just moving back to this account?

**Comment by samuelc02** [comment]
he seems to have changed his username to https://github.com/aricloverALT . But I'm also confused as to which repo is the main one, it seems this one has the most recent commits opposed to his alt, but in a previous post he kinda said he would focus more on the repo on the alt so idk. im also not sure which version is the latest being supported.

**Reply by yodaluca23** [reply]
Hmmm, I've been using this one, even when it was shadow banned... and as you said it's more up to date...
